build.gradle

```diff
@@ -53,6 +55,7 @@ project.ext.externalDependency = [
     'gmaTestModels': "com.linkedin.datahub-gma:test-models:$gmaVersion",
     'gmaTestModelsDataTemplate': "com.linkedin.datahub-gma:test-models-data-template:$gmaVersion",
     'gmaValidators': "com.linkedin.datahub-gma:validators:$gmaVersion",
+    'graphqlJava': 'com.graphql-java:graphql-java:13.0',
```

Comment:
Latest version seems to be 16.1  (https://github.com/graphql-java/graphql-java/releases)
Any reason why you picked 13.0?

Reply:
Good catch!!